api/src/main/java/org/geysermc/geyser/api/extension/ExtensionDescription.java

```diff
@@ -73,19 +80,21 @@ public interface ExtensionDescription {
     int minorApiVersion();
 
     /**
-     * Gets the extension's patch api version
-     *
-     * @return the extension's patch api version
+     * No longer in use. Geyser is now using an adaption of the romantic versioning scheme.
+     * See <a href="https://gist.github.com/Redned235/a24a593026c11308dbcf73ab6e9d07d1">here</a> for details.
```

Comment:
Commit this file (to the `api` repo i guess?) and reference it (targeting the master branch even though it won't exist yet)

Reply:
https://github.com/GeyserMC/Geyser/pull/3880/commits/c6301b3b03eeceede94090e0726ed0eaec4ddffd addressed here